app/models/county.rb

```diff
@@ -10,6 +10,11 @@ class County < ApplicationRecord
 
   validates :name, :fips_code, presence: true
 
+  enum rms: {
+    ocis: 'OCIS',
+    kp: 'KP'
+  }
+
```

Comment:
Is this a non-numeric enum? I'm going to start using this immediately. So much better than numeric mapping